src/libraries/Microsoft.Extensions.FileSystemGlobbing/src/Microsoft.Extensions.FileSystemGlobbing.csproj

```diff
@@ -1,23 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <RootNamespace>Microsoft.Extensions.FileSystemGlobbing</RootNamespace>
-    <AssemblyName>Microsoft.Extensions.FileSystemGlobbing</AssemblyName>
-    <Description>File system globbing to find files matching a specified pattern.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
If there is only 1 TFM, do we have to use `TargetFrameworks`? There is a non-zero cost to using `TargetFrameworks` when you only have a single TFM.

Reply:
Yes, we need to use TargetFramework in dotnet/runtime right now. https://github.com/dotnet/runtime/issues/31823

Reply:
That seems unfortunate. It has always been our public guidance to users to use `TargetFramework` when you only have a single TFM, and `TargetFrameworks` when you have multiple.

Reply:
It's a known issue, we'll get around to fixing it.  For now, it's expected that in dotnet/runtime this will be the case.  /cc @Anipik @ViktorHofer